readme-template.md

```diff
@@ -0,0 +1,107 @@
+# Repo Title
+
+<!-- Not all of these are essential but keep this in mind for the repo you are using -->
+![Build badge]() ![Test Coverage]() ![npm package]()
+
+<!--Short sentence on what the repo does-->
+
+<!-- possibly list experts in this repo for who to go to when encountering a problem-->
+
+##Contents
+* [Further Documentation](#documentation)
```

Comment:
I think it would be worth moving the location of the documentation after running locally as most of the time I mainly looking on how to set up the repo and get going. 

Do you think running locally needs its own section? 99% of our repos are going to be `npm run start`